src/jsp/jsp_cl.c

```diff
@@ -594,7 +594,7 @@ jsp_create_stored_procedure (PARSER_CONTEXT * parser, PT_NODE * statement)
   if (lang == SP_LANG_PLCSQL)
     {
       std::string pl_code (statement->sql_user_text, statement->sql_user_text_len);
-      err = plcsql_transfer_file (pl_code, false, compile_info);
+      err = plcsql_transfer_file (pl_code, true, compile_info);
```

Comment:
The verbose mode is only for debugging.
```suggestion
      err = plcsql_transfer_file (pl_code, false, compile_info);
```

Reply:
It can be benefitial to keep verbose mode for a few initial release versions of PL/CSQL and collect information from the customer sites obtained from the verbose mode logs.
Currently, the additional jobs done in the verbose mode are measuring times taken by compilations steps and logging them, whose overhead is expected to be little.

Reply:
Updated as suggested.